src/containers/Board/SymbolSearch/SymbolSearch.js

```diff
@@ -9,95 +9,41 @@ import messages from './messages';
 import mulberrySymbols from '../../../api/mulberry-symbols.json';
 import './SymbolSearch.css';
 
-let symbols = [];
-
-function renderInput(inputProps) {
-  const { home, value, ref, ...other } = inputProps;
-
-  return (
-    <TextField
-      fullWidth
-      autoFocus={home}
-      value={value}
-      inputRef={ref}
-      InputProps={{
-        ...other
-      }}
-    />
-  );
-}
-
-function renderSuggestion(suggestion, { query, isHighlighted }) {
-  return (
-    <div
-      className={classNames(
-        { 'SymbolSearch__Suggestion--highlighted': isHighlighted },
-        'SymbolSearch__Suggestion'
-      )}
-    >
-      <img
-        className="SymbolSearch__Suggestion-img"
-        src={suggestion.src}
-        alt=""
-      />
-      <div>
-        {suggestion.id}
-      </div>
-    </div>
-  );
-}
-
-function renderSuggestionsContainer(options) {
-  const { containerProps, children } = options;
-
-  return (
-    <div {...containerProps}>
-      {children}
-    </div>
-  );
-}
-
-function getSuggestionValue(suggestion) {
-  return suggestion.id;
-}
-
-function getSuggestions(value) {
-  const inputValue = value.trim().toLowerCase();
-  const inputLength = inputValue.length;
-  let count = 0;
-
-  return symbols.filter(symbol => {
-    if (count >= 16) {
-      return false;
-    }
-
-    let keep = symbol.id.slice(0, inputLength) === inputValue;
-
-    if (!keep) {
-      const words = symbol.id.split(' ');
-      for (let i = 1; i < words.length; i += 1) {
-        keep = words[i].slice(0, inputLength) === inputValue;
-        if (keep) {
-          break;
-        }
-      }
-    }
-
-    if (keep) {
-      count += 1;
-    }
-    return keep;
-  });
-}
-
 export class SymbolSearch extends Component {
   state = {
     value: '',
-    suggestions: []
+    suggestions: [],
+    symbols: []
```

Comment:
I would leave the symbols out of the state object, we can attach it directly to the instance of SymbolSearch - we want to keep the state to a minimum